### PR TITLE
added tests for sig_stars() function

### PR DESCRIPTION
**Pull request by Iqrar99**
Since the `sig_stars()` function in utils.py doesn't have tests. I added the tests to increase the coverage test.

### CODE REVIEW

**Comment by mwaskom** [comment]
Hm, I didn't actually realize this was still in the library. The only function that used it was removed many versions ago and it's not publicaly documented.

Tests are always good so once the quality control checks pass (your changes fail the style checker and one of your test cases is wrong) I can merge this PR, but I am also going to deprecate and plan to remove the `sig_stars` function.

File: seaborn/tests/test_utils.py [hunk 1]
```diff
@@ -105,6 +105,24 @@ def test_saturate():
...
+    stars2 = utils.sig_stars(0.001)
+    assert_equal(stars2, "**")
+    
+    stars3 = utils.sig_stars(0.01)
+    assert_equal(stars3, "*")
+    
+    stars4 = utils.sig_stars(0.09)
+    assert_equal(stars4, ".")
+    
+    stars5 = utils.sig_stars(0)
```
Comment (Iqrar99) [inline seaborn/tests/test_utils.py]: Ah yes, this should be 1

**Comment by Iqrar99** [comment]
ok, I will fix it now. I missed that 5th case. It should be 1

**Comment by Iqrar99** [comment]
hmm, I have no idea why I failed style checkers. Any suggestion? @mwaskom

**Comment by mwaskom** [comment]
Did you look at the Travis output? The relevant lines would be

```
W503,W504 --exclude seaborn/__init__.py,seaborn/colors/__init__.py,seaborn/cm.py,seaborn/external seaborn

seaborn/tests/test_utils.py:112:1: W293 blank line contains whitespace

seaborn/tests/test_utils.py:115:1: W293 blank line contains whitespace

seaborn/tests/test_utils.py:118:1: W293 blank line contains whitespace

seaborn/tests/test_utils.py:121:1: W293 blank line contains whitespace

seaborn/tests/test_utils.py:125:1: W293 blank line contains whitespace

Makefile:13: recipe for target 'lint' failed
```

**Comment by mwaskom** [comment]
fyi @lqrar99 check out this commit for a better way to write tests such as this: https://github.com/mwaskom/seaborn/pull/1973/files#diff-aa765aac3f5ebabf440c888c666886caL108

**Comment by Iqrar99** [comment]
Ahh I see. Thank you @mwaskom for the help 😊. I will learn from your codes.

**Pull request merged**